build_debian.sh

```diff
@@ -282,14 +277,16 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     makedumpfile            \
     conntrack               \
     python-pip              \
-    python3-pip
+    python3-pip             \
+    cron                    \
+    haveged
```

Comment:
Why is `haveged` needed?

Reply:
check the commit message here. https://github.com/Azure/sonic-buildimage/pull/4405/commits/2bdb348b92e5d0add128c1960d3d8f111e5466f8